packages/fast-components-react-base/src/button/button.props.ts

```diff
@@ -8,11 +9,15 @@ export enum ButtonHTMLTags {
     button = "button"
 }
 
-export interface IButtonProps {
+export interface IButtonHandledProps {
```

Comment:
@nicholasrice Can you ping the channel on this being a new pattern when this goes in?

Reply:
@chrisdholt yep